### PR TITLE
Style/fixes

### PR DESCRIPTION
**Pull request by Chiragmodi01**
### This pr includes responsiveness and ui fixes with few extra features
 Let me know if something in UI break or looks very messy both in desktop and mobile view

### CODE REVIEW

**Comment by aurobindoGupta** [comment]
- too many Elon at the top.
- if you are not logged in and go to watchlist or history there is a flicker where it checks if you logged in it shows the history page for 0.1 sec before displaying the login page fix it.
- preview of the video on hover is nice.
- change the theme dropdown menu to a toggle btn, dropdown seems unnecessary.

File: src/comps/FooterNav/FooterNav.jsx [hunk 1]
```diff
@@ -1,28 +1,44 @@
...
+  const {onMobile, userLoggedIn} = useMain()
+  let {pathname} = useLocation();
+  let navigate = useNavigate();
+
+  const IconHome = pathname === "/" ? AiFillHome : AiOutlineHome;
+  const IconPlaylist = pathname === "/playlist" ? MdVideoLibrary : MdOutlineVideoLibrary;
+  const IconWatchlater = pathname === "/watchlater" ? AiFillClockCircle : AiOutlineClockCircle;
+  const IconLike = pathname === "/liked" ? RiThumbUpFill : RiThumbUpLine;
+
+  const navigateToSignup = () => {
```
Comment (aurobindoGupta) [inline src/comps/FooterNav/FooterNav.jsx:17]: nice

File: src/comps/Navbar/Navbar.css [hunk 2]
```diff
@@ -85,8 +95,80 @@
...
+.searchbar {
+    position: relative;
+}
+.searchbar .search-suggestion-wrapper {
+    position: absolute;
+    width: calc(100% - 72px);
+    min-height: 5em;
+    max-height: 30rem;
+    height: max-content;
+    background-color: #ffffff;
```
Comment (aurobindoGupta) [inline src/comps/Navbar/Navbar.css:112]: variable

File: src/pages/SinglePageVideo/SinglePageVideo.css [hunk 3]
```diff
@@ -62,15 +65,24 @@
 .videoPage-video-title {
     font-size: 1.3rem;
 }
+.videoPage-left-main {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+    padding: 0 1.3rem;
```
Comment (aurobindoGupta) [inline src/pages/SinglePageVideo/SinglePageVideo.css:73]: oddly specific?

**Review by aurobindoGupta — APPROVED** [review]
nice work

**Review by harshitpaliwal95 — APPROVED** [review]
- great work chirag 
- cards comp looks really good 
- resposive design works great

**Pull request merged**